docs/pages/components/select.md

```diff
@@ -16,31 +16,52 @@ For lists that require more than 12 options, the <a href="/patterns/combobox-inp
 ## Sizes
 
 {% capture select-sizes %}
-<label class="fd-form-label">
+<label class="fd-form-label" id="f45564A57">
     Select Cozy Mode
 </label>
 <br>
 <div class="fd-popover">
   <div class="fd-popover__control">
      <div class="fd-select">
-         <div class="fd-select__control" role="button" tabindex="0" aria-controls="h0C6A325" aria-expanded="false" aria-haspopup="true">
+         <div
+            aria-controls="h0C6A325"
+            aria-expanded="false"
+            aria-haspopup="true"
+            class="fd-select__control"
+            role="button"
+            tabindex="0">
              Select
              <span class="fd-button fd-button--transparent sap-icon--slim-arrow-down fd-select__button"></span>
          </div>
       </div>
   </div>
-  <div class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--dropdown" aria-hidden="true" id="h0C6A325">
-     <ul class="fd-list fd-list--dropdown" role="listbox">
-        <li class="fd-list__item is-selected" aria-selected="true" role="option" tabindex="0">
+  <div
+    aria-hidden="true"
+    class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--dropdown"
+    id="h0C6A325">
+     <ul
+        aria-labelledby="f45564A57"
+        class="fd-list fd-list--dropdown"
+        role="listbox">
+        <li aria-selected="true"
+            class="fd-list__item fd-select__item is-selected"
+            role="option"
+            tabindex="0">
            <span class="fd-list__title">List item 1</span>
         </li>
-        <li class="fd-list__item" role="option" tabindex="-1">
+        <li class="fd-list__item fd-select__item"
+            role="option"
+            tabindex="-1">
            <span class="fd-list__title">List item 2</span>
        </li>
-        <li class="fd-list__item" role="option" tabindex="-1">
+        <li class="fd-list__item fd-select__item"
+            role="option"
+            tabindex="-1">
            <span class="fd-list__title">List item 3</span>
         </li>
-        <li class="fd-list__item" role="option" tabindex="-1">
+        <li class="fd-list__item fd-select__item"
+            role="option"
+            tabindex="-1">
```

Comment:
nothing changed in these - I was just spreading them out so I could see what I was doing. Happy to put them back if anyone is bothered by it